test/xpu_api/iterators/iterator.primitives/iterator.operations/advance.pass.cpp

```diff
@@ -19,50 +19,42 @@
 // template <RandomAccessIterator Iter>
 //   constexpr void advance(Iter& i, Iter::difference_type n);
 
-#include "oneapi_std_test_config.h"
-#include <CL/sycl.hpp>
-#include "test_iterators.h"
-#include "test_macros.h"
-#include <iostream>
+#include "support/test_config.h"
 
-#ifdef USE_ONEAPI_STD
-#    include _ONEAPI_STD_TEST_HEADER(iterator)
-namespace s = oneapi_cpp_ns;
-#else
-#    include <iterator>
-namespace s = std;
-#endif
+#include <oneapi/dpl/iterator>
 
-constexpr cl::sycl::access::mode sycl_read = cl::sycl::access::mode::read;
-constexpr cl::sycl::access::mode sycl_write = cl::sycl::access::mode::write;
+#include "support/test_iterators.h"
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+constexpr sycl::access::mode sycl_read = sycl::access::mode::read;
```

Comment:
1. It seems the constant `sycl_read `  is not used in the code...
2. regarding `sycl_write ` - I would like to use `sycl::access::mode::write` directly,  in just one place below where it needs.

Reply:
Fixed in all tests